src/middlewares/index.js

```diff
@@ -0,0 +1,47 @@
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import util from '../utils/response';
+
+const Util = new util()
+dotenv.config();
+const secretKey = process.env.JWT_SECRET_KEY;
+
+class Helper {
```

Comment:
I think helper function should be in the helper folder

Reply:
okay